dspace-xmlui/src/main/webapp/i18n/messages.xml

```diff
@@ -351,9 +351,9 @@
     <message key="xmlui.ArtifactBrowser.RestrictedItem.head_item_replaced">This item is replaced by a newer version</message>
     <message key="xmlui.ArtifactBrowser.RestrictedItem.para_item_replacedby">The requested item <b>{0}</b> is replaced by a new version which should be used instead. If you require this particular version, request it through our <a class="helpdesk-tolink">Help Desk</a>.</message>
 
-    <message key="xmlui.ArtifactBrowser.RestrictedItem.head_item_claimed">This item is managed by another repository</message>
-    <message key="xmlui.ArtifactBrowser.RestrictedItem.head2_item_claimed"><h4>You will find this item at <strong><span class="linkify">{0}</span></strong></h4></message>
-    <message key="xmlui.ArtifactBrowser.RestrictedItem.para_item_claimed">The authors asked us to hide this submission. <strong>We still keep all the data and metadata of the original submission</strong> but the submission is now located at <span class="linkify">{0}</span>. Please contact us at our <a class="helpdesk-tolink">Help Desk</a> if you need the original submission.</message>
+    <message key="xmlui.ArtifactBrowser.RestrictedItem.head_item_claimed"><h4>This item is managed by another repository</h4></message>
+    <message key="xmlui.ArtifactBrowser.RestrictedItem.para_item_claimed">You will find this item at the following location(s):</message>
+    <message key="xmlui.ArtifactBrowser.RestrictedItem.para2_item_claimed">The authors asked us to hide this submission. <strong>We still keep all the data and metadata of the original submission</strong> but the submission is now located at the above url(s). Please contact us at our <a class="helpdesk-tolink">Help Desk</a> if you need the original submission.</message>
```

Comment:
Please contact us at our <a class="helpdesk-tolink">Help Desk</a> if you need the original submission.
->
If you need the contents of the original submission, contact us at our <a class="helpdesk-tolink">Help Desk</a>.